pkg/registry/core/service/ipallocator/controller/repairip.go

```diff
@@ -165,29 +159,6 @@ func NewRepairIPAddress(interval time.Duration,
 		},
 	}, interval)
 
-	_, _ = serviceCIDRInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
hm... shouldn't we be re-validating IP addresses when ServiceCIDRs are deleted?

Reply:
the ServiceCIDR deletion is handled by the kube-controller-manager that must guarantee deleting a ServiceCIDR is safe and does not leave orphan ips

Reply:
in this controller the events were used to rebuild the iptree on changes